numpy/ma/core.pyi

```diff
@@ -337,7 +337,9 @@ fmod: _DomainedBinaryOperation
 mod: _DomainedBinaryOperation
 
 def make_mask_descr(ndtype): ...
-def getmask(a): ...
+
+def getmask(a: ArrayLike) -> NDArray[bool_] | bool_: ...
```

Comment:
it's probably not a common usecase, but scalar input always results in a `bool_`, which is overloadable with `numpy._typing._ScalarLike`.